style/hudl_sql_style_guide.md

```diff
@@ -45,9 +50,24 @@ description: A guide to writing clean, clear, and consistent SQL.
   __BAD__:
   `SELECT ksr.backings AS b`
 
-* Use [Common Table Expressions](http://www.postgresql.org/docs/8.4/static/queries-with.html) (CTEs) early and often, and name them well.
-* All dates should be formatted as 'YYYY-MM-DD'
-* Use `!=` for the inequality operator
+## `Common Table Expressions (CTEs)`
+
+* Use [Common Table Expressions](http://www.postgresql.org/docs/8.4/static/queries-with.html) often, and place them at the top
+* CTEs should be used to represent source tables to make code more readable
+* Only use full CTE names when referencing them in the SELECT statement
```

Comment:
maybe add `i.e. No aliases` so the reader knows where the full CTE names are meant to be used

Reply:
Per your suggestion added wording to make clearer.